packages/common/docs/API.md

```diff
@@ -245,10 +245,13 @@ isNil(undefined); // => true
 
 <a name="exp_module_util--exports.deprecate"></a>
 
-### exports.deprecate(name, version, [alternative]) ⏏
-Mark a piece of code as deprecated
+### exports.deprecate ⏏
+Mark a piece of code as deprecated.
 
-**Kind**: Exported function  
+Each deprecation notice for a given name and version combination will
+only be printed once.
+
+**Kind**: Exported member  
```

Comment:
Not sure why this is an "Exported member" now instead of an "Exported function". Yay `jsdoc2md`.